### ISSUE TITLE
Vagrant Setup - /srv/kubernetes missing from master

### ISSUE THREAD

**Issue by mbabker** [kubernetes/kubernetes#11813]
Local setup:
- OS X 10.10.4
- Vagrant 1.7.4
- VirtualBox 4.3.30
- Kubernetes 1.0.1 Release

Following the http://kubernetes.io/v1.0/docs/getting-started-guides/vagrant.html setup guide to boot a local instance via Vagrant, the provision scripts do not complete.  The script exits shortly after the `vagrant up` command completes and it appears to be due to a missing `/srv/kubernetes` folder on the master box.

Below screencap shows the last few lines of the provisioning output and a check that shows the `/srv/kubernetes` folder does not exist.

<img width="1032" alt="screen shot 2015-07-24 at 11 49 42 am" src="https://cloud.githubusercontent.com/assets/368545/8878684/791dd5c6-31fa-11e5-8259-1fcf9d3fa0c3.png">

When the first `vagrant ssh master` command executes after the `vagrant up` command finishes, an empty `kubecfg.crt` is written to my local `/tmp` folder then the script appears to exit as the next two commands don't seem to execute.  Below screen cap shows the `/tmp` directory contents.

<img width="724" alt="screen shot 2015-07-24 at 11 51 41 am" src="https://cloud.githubusercontent.com/assets/368545/8878699/83e93f40-31fa-11e5-8d9f-1310dfbe465a.png">

**Comment by mikedanese** [comment]
@jayunit100 @derekwaynecarr

**Comment by jayunit100** [comment]
If this effects mounting of certs then Could be related to the failing nettest as well.

**Comment by jmccarty3** [comment]
I was able to resolve the issue by commenting out lines 267-269 274-276 in cluster/vagrant/util.sh

I believe those lines are left over from before kubectrl was being used. The vagrant ssh commands exit due to error. You will also run into error described in #11819  after its set up.

**Comment by derekwaynecarr** [comment]
The issue with the network not being in a valid state was because of an error in the NetworkManager configuration for eth1.

Can you verify you are running with the following (which was cherry picked):
https://github.com/GoogleCloudPlatform/kubernetes/pull/12237

**Comment by mbforbes** [comment]
Friendly ping @mbabker re: https://github.com/kubernetes/kubernetes/issues/11813#issuecomment-129589034

**Comment by mbabker** [comment]
With the 1.0.3 release (which has #12237 included) the provisioning script gets past the point it was failing before but still has issues it seems.  The `/srv/kubernetes` folder does get created and `kube-up` proceeds to call `verify-cluster`, but now I get into an endless loop on the `Waiting for each minion to be registered with cloud provider` check.

I've got a build from master on commit 6afbaf6bf6ac8d1d71307819f21d56d7946c522c where I also tested deploying a Vagrant cluster.  In this case the `verify-cluster` step didn't get stuck in the loop and the cluster came up as expected.
